packages/neuron-wallet/src/database/chain/migrations/1684488676083-TxLock.ts

```diff
@@ -1,15 +1,15 @@
-import {MigrationInterface, QueryRunner, TableIndex} from "typeorm";
+import { MigrationInterface, QueryRunner, TableIndex } from 'typeorm'
 
 export class TxLock1684488676083 implements MigrationInterface {
-
-    public async up(queryRunner: QueryRunner): Promise<void> {
-      await queryRunner.query(`CREATE TABLE "tx_lock" ("lockHash" varchar NOT NULL, "transactionHash" varchar NOT NULL, PRIMARY KEY ("transactionHash", "lockHash"))`)
-      await queryRunner.createIndex("tx_lock", new TableIndex({ columnNames: ['lockHash'] }))
-      await queryRunner.createIndex("tx_lock", new TableIndex({ columnNames: ['transactionHash'] }))
-    }
-
-    public async down(queryRunner: QueryRunner): Promise<void> {
-      await queryRunner.query(`DROP TABLE "tx_lock"`)
-    }
-
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "tx_lock" ("lockHash" varchar NOT NULL, "transactionHash" varchar NOT NULL, PRIMARY KEY ("transactionHash", "lockHash"))`
+    )
+    await queryRunner.createIndex('tx_lock', new TableIndex({ columnNames: ['lockHash'] }))
```

Comment:
During the discussion on formatting, it was discovered that the index column was changed from `transactionHash` to `lockHash`, we should NOT modify a published migration file

Reply:
sorry, I misread this line, it does not modify the index, it is the wrong mark on the diff view